pkg/verifier/gcp/gcp_verifier.go

```diff
@@ -259,7 +259,8 @@ func generateUserData(variables map[string]string) (string, error) {
 	variableMapper := func(varName string) string {
 		return variables[varName]
 	}
-	data := os.Expand(helpers.UserdataTemplate, variableMapper)
+	// TODO: REPLACE JUNK VALUE "helpers.UserdataTemplate" BELOW
+	data := os.Expand("helpers.UserdataTemplate", variableMapper)
```

Comment:
Note: this change is make the compiler happy. GCP support is already broken, so this logical "non-fix" doesn't actually take anything away.